packages/cli/src/index.ts

```diff
@@ -70,7 +68,7 @@ sourceMap.install();
 Sentry.init({
   dsn: SENTRY_DSN,
   release: `vercel-cli@${pkg.version}`,
-  environment: isCanary ? 'canary' : 'stable',
+  environment: 'stable',
```

Comment:
Perhaps we should have a similar concept for versions auto published from PRs. Basically detect if the version was installed from npm or from a tarball.

Reply:
We already get that information from `release` via `pkg.version`. Perhaps we should just remove this tag entirely?

Reply:
SGTM. Let's do it in a follow-up though to keep this one "clean" from behavior changes.

Reply:
I actually think its better to do it in this PR because most of the same lines would be changed

Reply:
The only place that makes sense to me to do it would be this line and perhaps the feedback one below.